lib/MySQL_Thread.cpp

```diff
@@ -475,6 +475,7 @@ static char * mysql_thread_variables_names[]= {
 	(char *)"query_digests_track_hostname",
 	(char *)"query_digests_keep_comment",
 	(char *)"parse_failure_logs_digest",
+	(char *)"log_mysql_statements",
```

Comment:
As a general rule in MySQL, we prefer to put new additions only in the end of the array. I hope that is not the case here.